scripts/microbe_stage/microbe_editor/microbe_editor.as

```diff
@@ -10,13 +11,18 @@
 ////////////////////////////////////////////////////////////////////////////////
 */
 
+bool freeBuilding = false;
```

Comment:
This should be next to the other variables, if this is kept here. However I think that something like "has done freebuild" should be placed in the PlayerData class and the editor should check that when entering. Also that should disable winning. The cleanest way would probably to use PlayerData::setBool and the isBoolSet methods (they allow us to have a bunch of flags related to the current game without adding new methods all the time, and saving them is probably easier).